File: HelloWorld/app/src/main/java/com/example/helloworld/MainActivity.java
package com.example.helloworld;

import android.content.Context;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);

        Button button = (Button)findViewById(R.id.button);
        EditText editText = (EditText)findViewById(R.id.editTextText);

        if(button != null)
        {
            button.setOnClickListener((View.OnClickListener)(new View.OnClickListener()
            {
                        public final void onClick(View it)
                        {
                            if(editText.length() == 0) //null
                            {
                                // displaying a toast message
                                Toast.makeText((Context)MainActivity.this, "Hello World Testing", Toast.LENGTH_LONG).show();
                            } else {
                                // displaying a toast message
                                Toast.makeText((Context)MainActivity.this, (CharSequence) editText.getText(), Toast.LENGTH_LONG).show();
                            }
                        }
            }));
        }
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;
        });
    }
}

File: HelloWordKotlin/app/src/main/java/com/project/hellowordkotlin/MainActivity.kt
package com.project.hellowordkotlin

import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.TextView
import android.widget.Toast
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import com.google.android.material.textfield.TextInputLayout

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }
    }

    fun clickOnMe(view: View) {
        Toast.makeText(applicationContext, "Hello World Button!!", Toast.LENGTH_SHORT).show()
    }

    fun onRegister(view: View) {
        val myTextView = findViewById<TextView>(R.id.emailId)
        val myEmailTextView = findViewById<TextView>(R.id.emailTextID)

        val emailAdress:String = myTextView.text.toString()
        Toast.makeText(applicationContext, "Register Button!!\n" + emailAdress, Toast.LENGTH_SHORT).show()

        myEmailTextView.text = myTextView.text.toString()
    }

    fun onLogin(view: View) {
        Toast.makeText(applicationContext, "Login Button!!", Toast.LENGTH_SHORT).show()
    }
}